src/main/groovy/life/qbic/portal/sampletracking/web/presenters/SampleImportPresenter.groovy

```diff
@@ -0,0 +1,40 @@
+package life.qbic.portal.sampletracking.web.presenters
+
+import life.qbic.datamodel.samples.Sample
+import life.qbic.portal.sampletracking.trackinginformation.list.ModifySampleListOutput
+import life.qbic.portal.sampletracking.web.ViewModel
+
+/**
+ * <add class description here>
+ *
+ * @author: Sven Fillinger
+ *
+ */
+class SampleImportPresenter implements ModifySampleListOutput{
+
+    final ViewModel viewModel
```

Comment:
```suggestion
    final private ViewModel viewModel
```
This can be made private since this field is not of interest outside this class.